modules/customize-certificates-understanding-service-serving.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/service-serving-certificate.adoc
+
+[id="understanding-service-serving-{context}"]
+= Understanding service serving certificate secrets
+
+Service serving certificate secrets are intended to support complex 
+middleware applications that require encryption. These certificates have 
+the same settings as the server certificates that are generated by the 
```

Comment:
I think my previous comment on this one got eaten when I refreshed the PR..
We should just say that the server certificates are issued as a TLS Web Server certificate.

 I don't think that the part about administrator tooling is relevant for 4.

Reply:
I don't think the comment was eaten! This feedback has been included.